doc/source/support/frequently-asked-questions.rst

```diff
@@ -134,6 +134,44 @@ Achieving the same result in Markdown often requires raw HTML, which is less rea
 
 Switching to ``README.rst`` at Level 5 helps users appreciate the formatting power of ``.rst`` and serves as a stepping stone toward writing full documentation in ``.rst`` as part of the ``scikit-package`` documentation standard.
 
+.. _faq-set-default-prompt-value:
+
+How can I change the default values that appear in the prompt when creating projects in level 3,4,5?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can override the existing default values in the prompt by creating and editing ``~/.skpkgrc``. For example, to change some default values prompted when creating a new package in level 5, please follow the quick steps below:
```

Comment:
We all know what ~ means here but let's be explicit in the docs.  This is a point of failure because if someone doesn't know this they the are 100% stuck at this point.  Say explicitly something like "`.skpkgrc` in the users `home` directory.  In `bash` your home directory is designated by `~`, so the commands
```
cd ~
touch .skpkgrc
edit .skpkgrc
```
will move you to the home directory, create the file and open it in your default editor.  You can also do these tasks using Explorer/Finder and using a windows editor."

something like that?

Reply:
@ycexiao pls comment